src/Core/src/PropertyMapper.cs

```diff
@@ -169,12 +222,22 @@ public void Add(string key, Action<TViewHandler, TVirtualView> action) =>
 			SetPropertyCore(key, (h, v) =>
 			{
 				if (v is TVirtualView vv)
+				{
 					action?.Invoke((TViewHandler)h, vv);
+				}
 				else if (Chained != null)
 				{
 					foreach (var chain in Chained)
```

Comment:
Just noticing something here, this block iterates over the Chained and then calls TryUpdatePropertyCore which calls MergedMappers which calls SnapshotMappers which calls GetKeys which also iterates over the Chained. This may be the super late night review here, but will it in some cases loop Chained^2?

Reply:
At this point in time, `MergedMappers` will be already populated, so that won't happen.
Even if it happens I don't see any risk involved.